Aardvark.podspec

```diff
@@ -1,11 +1,12 @@
 Pod::Spec.new do |s|
   s.name     = 'Aardvark'
-  s.version  = '3.4.0'
+  s.version  = '3.4.1'
```

Comment:
going with a bug fix change since we're still supporting version 3.0, despite what our podspec says below.